knowledge-base/loader-container-center-in-viewport.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Center LoaderContainer in viewport
+description: How to center the LoaderContainer in viewport
+type: how-to
+page_title: Center LoaderContainer in viewport
```

Comment:
page_title can be just a little longer to add 1-2 more keywords. If you think "page" will be more relevant, consider that instead - the customer used it as well.

```suggestion
page_title: Center LoaderContainer in the Browser Viewport
```